src/main/scala/slamdata/SbtSlamdata.scala

```diff
@@ -73,7 +82,7 @@ object SbtSlamData extends AutoPlugin {
         "Scalaz Bintray Repo" at "http://dl.bintray.com/scalaz/releases",
         "bintray/non" at "http://dl.bintray.com/non/maven"),
       addCompilerPlugin("org.spire-math"  %% "kind-projector" % "0.9.3"),
-      addCompilerPlugin("org.scalamacros" %  "paradise"       % "2.1.0" cross CrossVersion.full),
+      addCompilerPlugin("org.scalamacros" %  "paradise"       % "2.1.0" cross CrossVersion.fullMapped(stripTLSVersion)),
```

Comment:
The correct fix here is just to change `CrossVersion.full` to `CrossVersion.patch`.